docs/Concepts/Data-Storage-Formats.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Data storage formats
+---
+
+# Data storage formats
+
+Besu offers two formats for storing the world state, [Forest of Tries](#forest-of-tries) and [Bonsai Tries](#bonsai-tries).
+
+## Forest of Tries
+
+Forest of Tries, also called forest mode, is the [default storage format](../HowTo/Configure/Configure-Data-Storage.md).
```

Comment:
```suggestion
Besu offers two formats for storing the world state, [Forest of Tries](#forest-of-tries) and [Bonsai Tries](#bonsai-tries).
You can [configure the data storage format](../HowTo/Configure/Configure-Data-Storage.md) when starting Besu.

## Forest of Tries

Forest of Tries, also called forest mode, is the default storage format.
```

I think it makes more sense to link to the how-to information under the general heading.